### DISCUSSION TITLE
Release the Experimental version in the installer again.

### DISCUSSION THREAD

**Discussion by Silvaronn** [flybywiresim/aircraft#6349] [Ideas]
I see the exp version is getting constant updates, release it for us. It is more intuitive and easier than downloading directly from the website. If this is not possible, do you have any plans to release it again? TKS!!

**Comment by GhostEagle68** [comment]
> I see the exp version is getting constant updates, release it for us. It is more intuitive and easier than downloading directly from the website. If this is not possible, do you have any plans to release it again? TKS!!

Experimental is on hold until developers can agree that it's usable for public testing of new features and fixes. Right now it's not recommended for public use.
